docs/docs/core/entity-service.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Entity Service
+slug: /entity-service
```

Comment:
IMO this should be `core/database/entity-service` because afaik it relates to the `database` package. wdyt? 

cc @Marc-Roig

Reply:
helloo, 
the entity service lives in core actually , not in the database package
`packages/core/strapi/lib/services/entity-service/index.js`

Reply:
ah okay, so maybe it should be `core/strapi/entity-service`? because it still lives in a package?

Reply:
indeed! it could be in `core/strapi/entity-service`

Reply:
`core/strapi/entity-service` it is

Reply:
@joshuaellis Then we should probably do the same for the event-hub

Reply:
If you could move it around, i'd be really grateful :)

Reply:
I moved the location of the file to core/strapi/entity-service
Changing the slug breaks breaks the sidebar and I can't fix that since the location would be 
`core/strapi/entity-service/core/strapi/entity-service` what does not work